lib/common/zstdmt_depthreadpool.h

```diff
@@ -0,0 +1,45 @@
+#ifndef ZSTDMT_DEPTHREADPOOL_H_MODULE
+#define ZSTDMT_DEPTHREADPOOL_H_MODULE
+
+#include <stddef.h>
+
+/***************************************
+* Definitions
+***************************************/
+
+typedef void (*ZSTDMT_depThreadPoolFn)(void* data);                          /* Job callback function */
+typedef struct ZSTDMT_depThreadPoolJob_s ZSTDMT_depThreadPoolJob_t;          /* Job management struct */
```

Comment:
minor : naming style : 
types should have the first letter in Capital.
Also, try to avoid suffix `*_t` when possible.
So here it would be : ZSTDMT_DepThreadPoolJob.

We have not been very consistent in enforcing these rules in the past, so you can find plenty of counter-examples in the code base. Let's try to make some progresses now.